builtin/common/settings/dlg_settings.lua

```diff
@@ -615,7 +615,13 @@ local function get_formspec(dialogdata)
 		local info_reset_y = used_h / 2 - 0.25
 
 		if show_reset then
-			local default = comp.setting.default
+			local default
```

Comment:
Missing fallback for `comp.setting.type ~= "key"`:

```suggestion
			local default = comp.setting.default
```

(Then you can also replace `comp.setting.default` in the `if` block below with just `default`.)

Reply:
One should not code at 10:30 PM lol.
I know how to code Lua, but I don't know how to code Lua _**well**_ XD